server/src/services/metadata.service.ts

```diff
@@ -411,7 +411,11 @@ export class MetadataService {
       }
       const checksum = this.cryptoRepository.hashSha1(video);
 
-      let motionAsset = await this.assetRepository.getByChecksum(asset.libraryId ?? null, checksum);
+      let motionAsset = await this.assetRepository.getByChecksum({
+        ownerId: asset.ownerId,
+        libraryId: asset.libraryId ?? undefined,
```

Comment:
```suggestion
        libraryId: asset.libraryId,
```

Reply:
It has a null type but the method doesn't accept null

Reply:
Ah shoot I'm stupid, never mind.